src/Sarif.Driver/Sdk/RuntimeConditions.cs

```diff
@@ -39,6 +39,7 @@ public enum RuntimeConditions
         NoValidAnalysisTargets = 0x400,
         RuleMissingRequiredConfiguration = 0x800,
         TargetParseError = 0x1000,
+        ExceptionInvalidCommandLineOption = 0x2000,
```

Comment:
Why isn't this just 'InvalidCommandLineOption'? Isn't that the runtime condition of note?